backend/server/src/services/tournamentService.ts

```diff
@@ -88,6 +88,183 @@ export class TournamentService {
     return await newTournament.toObject();
   }
 
+  public async addTeamToTournament(
+    tournamentId: string,
+    teamName: string,
+    creatorId: string
+  ): Promise<Tournament> {
+    const tournament = await TournamentModel.findById(tournamentId);
+
+    if (tournament === null || tournament === undefined) {
```

Comment:
`if (!tournament)` does the trick for future notice. I see few of these in this pr but wont comment all since no need to necessarily "fix"

Reply:
The GitHub Lint check gave the error that it might be null, so I had to do that. Initially it was !tournament